node/src/actors/chain_manager/mining.rs

```diff
@@ -126,6 +129,29 @@ impl ChainManager {
         let own_pkh = self.own_pkh.unwrap_or_default();
         let is_ars_member = rep_engine.is_ars_member(&own_pkh);
 
+        let superblock_period = u32::from(chain_info.consensus_constants.superblock_period);
+        // FIXME: Only ARS members from the last block of the previous SuperBlock can create a SuperBlock
+        if rep_engine.is_ars_member(&own_pkh) && current_epoch % superblock_period == 0 {
+            // FIXME(#1236): ARS Members have to include the BLS signature instead of PublicKeyHash
+            // FIXME: After Reputation Merkelitation, only the ARS Members from the previous block
+            // consolidated will be added, to avoid include addresses that could be wrong later by
```

Comment:
```suggestion
            // FIXME: After Reputation Merkelitation, only the ARS Members from the previous consolidated
            // block will be added, to avoid include addresses that could be wrong later by
```